common/security-features/subresource/subresource.py

```diff
@@ -1,4 +1,5 @@
-import os, json, urllib, urlparse
+import os, json
+from six.moves.urllib.parse import parse_qsl, SplitResult, urlencode, urlsplit,  urlunsplit
```

Comment:
`SplitResult` isn't obviously used here, is it necessary to import?

Reply:
It is used at line 79  in common/security-features/subresource/subresource.py

Reply:
Ah, my mistake, sorry!